### PR TITLE
feat: standardization of designs in website

### PR DESCRIPTION
**Pull request by sahas-01**
The following changes have been made:
- standardization of pages as per the updated design system - with font sizes, weights, padding issues, etc.
- detailed listing view page has also been updated with the above mentioned changes
- border radius has been updated in certain places

### CODE REVIEW

File: src/components/Listing/Tracks/TrackListingCard.tsx [hunk 1]
```diff
@@ -102,25 +107,21 @@ const TrackListingCard = ({ className, posts, trackName } : Props) => {
 		handlePaginationChange({ limit: LISTING_LIMIT, page });
 	};
 	return (
-		<div className={`${className} bg-white drop-shadow-md rounded-md p-4 md:p-8 text-sidebarBlue `}>
-			<div className='flex items-center justify-between mb-10'>
-				<div>
-					<h1 className='dashboard-heading'>Referenda</h1>
```
Comment (KanishkaRajputd) [inline src/components/Listing/Tracks/TrackListingCard.tsx:108]: why did you remove FilteredTags?

Comment (KanishkaRajputd) [reply src/components/Listing/Tracks/TrackListingCard.tsx:108]: and replace it with FilterByTags

File: src/ui-components/OnchainCreationLabel.tsx [hunk 2]
```diff
@@ -41,14 +40,7 @@ const OnchainCreationLabel = ({ address, topic, username, truncateUsername }:Pro
 						: null
 				}
 			</div>
-			{
-				topic?
-					<div className='flex items-center'>
-						<div className='mr-1.5 ml-auto hidden min-[340px]:flex'>from</div>
-						<TopicTag topic={topic} />
```
Comment (KanishkaRajputd) [inline src/ui-components/OnchainCreationLabel.tsx:48]: TopicTag why we remove

File: src/components/Listing/FellowshipReferendum/TrackListingCard.tsx [hunk 3]
```diff
@@ -8,7 +8,6 @@ import React, { FC, useState } from 'react';
 import TrackListingAllTabContent from './TrackListingAllTabContent';
 import TrackListingTabContent from './TrackListingTabContent';
 import FilterByTags from '~src/ui-components/FilterByTags';
-import FilteredTags from '~src/ui-components/filteredTags';
```
Comment (KanishkaRajputd) [inline src/components/Listing/FellowshipReferendum/TrackListingCard.tsx:11]: filteredTags ? why did you remove it

File: src/components/Listing/Tracks/TrackListingCard.tsx [hunk 4]
```diff
@@ -13,9 +13,9 @@ import CountBadgePill from '~src/ui-components/CountBadgePill';
 import TrackListingAllTabContent from './TrackListingAllTabContent';
 import TrackListingStatusTabContent from './TrackListingStatusTabContent';
 import FilterByTags from '~src/ui-components/FilterByTags';
-import FilteredTags from '~src/ui-components/filteredTags';
```
Comment (KanishkaRajputd) [inline src/components/Listing/Tracks/TrackListingCard.tsx:16]: why did you remove filteredTags?

File: pages/_app.tsx [hunk 5]
```diff
@@ -20,7 +20,6 @@ import { NetworkContextProvider } from '~src/context/NetworkContext';
 import getNetwork from '~src/util/getNetwork';
 
 export const poppins = Poppins({
-	adjustFontFallback: false,
```
[the remark this answers is not in the document]
Comment (sahas-01) [reply pages/_app.tsx:23]: Added, pls check

File: src/ui-components/OnchainCreationLabel.tsx (hunk 2 above)
Comment (sahas-01) [reply src/ui-components/OnchainCreationLabel.tsx:48]: TopicTag added in a different place to align with designs

File: pages/parachains/index.tsx [hunk 6]
```diff
@@ -68,12 +68,12 @@ const Parachains = ({ className, network }: Props) => {
 					</Col>
 				</Row>
 
-				<div className={`${className} bg-white drop-shadow-md p-2 lg:p-6 rounded-md h-[650px]`}>
-					<h2 className='dashboard-heading mb-6'>Projects</h2>
+				<div className={`${className} bg-white drop-shadow-md p-2 lg:p-6 rounded-xl h-[650px]`}>
+					<h2 className='text-bodyBlue text-xl font-medium leading-8 mb-6 mt-2 sm:mt-0'>Projects</h2>
 					<Tabs
-						tabBarStyle={{ color:'#334D6E' }}
+						tabBarStyle={{ color:'#243A57' }}
```
Comment (alphainfinitus) [inline pages/parachains/index.tsx:74]: arbitrary values here for `tabBarStyle` and `className` here

File: pages/parachains/index.tsx [hunk 7]
```diff
@@ -122,6 +122,11 @@ export default styled(Parachains)`
 		}
 	}
 
+	.ant-table-thead{
+		color: #243A57 !important;
```
Comment (alphainfinitus) [inline pages/parachains/index.tsx:126]: arbitrary value here

File: src/components/Gov2Home/Gov2LatestActivity/AllGov2PostsTable.tsx [hunk 8]
```diff
@@ -33,11 +33,11 @@ const columns: ColumnsType<IPostsRowData> = [
 		render: (title) => {
 			return (
 				<>
-					<h4
+					<div
```
Comment (alphainfinitus) [inline src/components/Gov2Home/Gov2LatestActivity/AllGov2PostsTable.tsx:36]: Please use heading tags instead of divs for titles for SEO

File: src/components/Gov2Home/Gov2LatestActivity/TrackPostsTable.tsx [hunk 9]
```diff
@@ -44,7 +44,7 @@ const columns: ColumnsType<IPostsRowData> = [
 		title: 'Posted By',
 		dataIndex: 'username',
 		key: 'postedBy',
-		render: (username, { proposer }) => <NameLabel textClassName='max-w-[9vw] 2xl:max-w-[12vw]' defaultAddress={proposer} username={username} disableIdenticon={false} />
+		render: (username, { proposer }) => <NameLabel textClassName='text-[#243A57] max-w-[9vw] 2xl:max-w-[12vw]' defaultAddress={proposer} username={username} disableIdenticon={false} />
```
Comment (alphainfinitus) [inline src/components/Gov2Home/Gov2LatestActivity/TrackPostsTable.tsx:47]: arbitrary colour in `textClassName`

File: src/components/GovernanceCard.tsx [hunk 10]
```diff
@@ -83,107 +85,194 @@ const GovernanceCard: FC<IGovernanceProps> = (props) => {
...
+							<div className='xs:hidden sm:flex items-center justify-center gap-x-1.5 mr-0.5'>
+								<DislikeOutlined style={{ color: '#485F7D' }} />
+								<span className='text-lightBlue'>{getFormattedLike(postReactionCount['👎'])}</span>
+							</div>
+							{
+								isCommentsVisible?
+									<>
+										<div className='xs:hidden text-lightBlue sm:flex items-center'>
+											<NewChatIcon style={{
+												color: '#485F7D'
```
Comment (alphainfinitus) [inline src/components/GovernanceCard.tsx:129]: arbitrary value here

File: src/components/GovernanceCard.tsx [hunk 11]
```diff
@@ -83,107 +85,194 @@ const GovernanceCard: FC<IGovernanceProps> = (props) => {
...
+									: null
+							}
+						</div>
 
-					{!!end && !!currentBlock &&
-							<div className="flex items-center">
-								<Divider className='hidden lg:inline-block' type="vertical" style={{ borderLeft: '1px solid #90A0B7' }} />
+						{!!end && !!currentBlock &&
+							<div className="flex text-lightBlue items-center">
+								<Divider className='hidden lg:inline-block' type="vertical" style={{ borderLeft: '1px solid #485F7D' }} />
```
Comment (alphainfinitus) [inline src/components/GovernanceCard.tsx:170]: arbitrary value here

**Review by alphainfinitus — APPROVED** [review]
